src/core/registration.h

```diff
@@ -21,6 +21,17 @@ typedef enum QUIC_CONNECTION_ACCEPT_RESULT {
     QUIC_CONNECTION_REJECT_APP
 } QUIC_CONNECTION_ACCEPT_RESULT;
 
+typedef enum QUIC_REGISTRATION_REF {
+
+    QUIC_REGI_REF_HANDLE_OWNER,         // Application or Core.
```

Comment:
The `QUIC_REGI_` prefix seems novel - I'd recommend the full `QUIC_REGISTRATION_`, or otherwise truncate to 3 characters (`QUIC_REG_`), which is a Windows/NT style norm.